labs/8.md

```diff
@@ -1,9 +1,103 @@
-## Conclusion
+# Convert a Namespace to a Federated Namespace
+Applications within OpenShift may already be deployed in an unfederated way. The `kubefedctl` tool will allow for a namespace and the objects within that namespace to become federated objects. This lab will deploy an application with a deployment and service and then use kubefedctl to federate the application and have it running on all three clusters. 
 
-Thanks for attending this hands on lab. By now you have completed the following activities.
+## Create a Namespace
+To start with we will create a namespace for the *reverse-words* application to be deployed.
 
-* Explored the environment
-* Deployed KubeFed
-* Flattened out your configuration files
-* Deployed 2 applications
-* Etc...
+~~~sh
+oc create ns --context cluster1 reverse-words
+oc project --context cluster1 reverse-words
+~~~
+
+Now that the namespace exists deploy the deployment and service objects.
+~~~sh
+oc create --context cluster1 -f 01-service.yaml
```

Comment:
In case "oc project" is not used, add "-n reverse-words" to each command, otherwise leave it as it's now.